.travis.yml

```diff
@@ -1,25 +1,25 @@
 language: python
+python:
+  - 3.4
+  - 3.6
+dist: precise
+
+services:
+  - docker
 
 matrix:
+  fast_finish: true
+
+stages:
+  - lint
+  - test
```

Comment:
This "test" stage is some sort of default? I don't see it referenced in a job later on

Reply:
yes, it's the default stage. I need to declare the order here because by default stages are run after the test one.